src/Transformer/FractalTransformer.php

```diff
@@ -76,7 +76,9 @@ public function transform($response, $transformer, Binding $binding, Request $re
         }
 
         if ($response instanceof EloquentCollection) {
-            $response->load($this->fractal->getRequestedIncludes());
+            $requestedIncludes = $this->fractal->getRequestedIncludes();
+            $eagerLoads = $this->getEagerLoads($transformer, $requestedIncludes);
+            $response->load($eagerLoads);
```

Comment:
Well, it shouldn't. Besides, I can't imagine why you wouldn't want to lazily eager load them. Saves running the loading yourself or loading them individually.